### PR TITLE
Configure certificates via Helm values

### PR DESCRIPTION
**Pull request by dereulenspiegel**
As there are users who configure certificates via Helm values (i.e. via Helm secrets) it would be nice to have the ability to configure these certificates without needing to create a new Chart with the vernemq as a subchart. This PR adds that under the `certificates` config key in the Helm values.
This PR should not change the default behavior if the key is not set or empty.

### CODE REVIEW

File: helm/vernemq/values.yaml [hunk 1]
```diff
@@ -231,12 +233,6 @@ additionalEnv:
 #    value: "10000"
 #  - name: DOCKER_VERNEMQ_MAX_OFFLINE_MESSAGES
 #    value: "-1"
-#  - name: DOCKER_VERNEMQ_LISTENER__SSL__CAFILE
```
Comment (ioolkos) [inline helm/vernemq/values.yaml:234]: @dereulenspiegel can you please keep this comment in, so that we have "the old ways" still documented.

**Review by ioolkos — CHANGES_REQUESTED** [review]

File: helm/vernemq/values.yaml (hunk 1 above)
Comment (dereulenspiegel) [reply helm/vernemq/values.yaml:234]: @ioolkos Of course. But maybe it makes sense add a little comment warning of potential conflicts with the `certificate` config key?

Comment (ioolkos) [reply helm/vernemq/values.yaml:234]: jep, sure, feel free to add that in place there. Thanks!

Comment (dereulenspiegel) [reply helm/vernemq/values.yaml:234]: I readded the removed example env config and added a little comment.

**Review by ioolkos — DISMISSED** [review]

**Pull request merged**